uahf-technical-spec.md

```diff
@@ -208,6 +219,15 @@ digest based on BIP143. Revisions were made to account for non-Segwit
 deployment.
 
 
+### REQ-6-4 (mandatory use of SCRIPT_VERIFY_STRICTENC)
+
+Once the fork has activated, a transaction shall be deemed invalid if
```

Comment:
@zander is correct. This flag is not set on the transaction, but passed to the script verification machinery.